share/completions/dpkg-reconfigure.fish

```diff
@@ -8,7 +8,7 @@ complete -x -f -c dpkg-reconfigure -s h -l help     --description 'Display help'
 # General options
 complete -f -c dpkg-reconfigure -s f -l frontend -r -a "dialog readline noninteractive gnome kde editor web" --description 'Set configuration frontend'
 complete -f -c dpkg-reconfigure -s p -l priority -r -a "low medium high critical" --description 'Set priority threshold'
-complete -f -c dpkg-reconfigure -l default-priority --description 'Use default priority threshold'
+complete -f -c dpkg-reconfigure -l default-priority --description "Use current default ("(echo get debconf/priority | debconf-communicate | string match -r '\w*$')") priority threshold"
```

Comment:
> That'll only work if it only ever prints one thing.

I assume that the output always consists of two tokens, but I agree that `\w+$` is better than `\w*$`. Indexing with `[-1]` would be the same as matching `\w+$`, wouldn't it? I like the indexing a bit more, but I'd go with that regex if you prefer.